src/days-2-4-adv/adv.py

```diff
@@ -49,3 +83,53 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+def help_prompt():
+    print('Hello, travelers. Here are the available commands: \n')
+    print('Move:')
+    print('  North: n or forward \n  South: s or backwards \n  East: e or right \n  West: w or left')
+    print('Look:')
+    print('  Look (direction) to look at what lies ahead.')
+    print('Items:')
+    print('  Get/take (item) to take item \n  Drop (item) to drop item \n  i or inventory to check inventory')
+    print('Help:')
+    print('  h or help to view these commands again')
+    print('Quit:')
+    print('  q to quit the game')
+
+
+valid_directions = {'n': 'n', 's': 's', 'e': 'e', 'w': 'w',
+                    'forward': 'n', 'backwards': 's', 'right': 'e', 'left': 'w'}
+valid_utilities = {'i': 'i', 'inventory': 'i'}
+valid_help = {'h': 'h', 'help': 'h'}
+
+p = Player(input('What is your name? \n'), room['outside'])
+print(p.room)
+p.room.room_items()
+print('\n\nPress h or help to view commands.')
+
+while True:
+    cmd = input('-> ').lower().split(' ')
+    if len(cmd) == 1:
+        if cmd[0] == 'q':
```

Comment:
how about a print message to let the user know the game intentionally ended? maybe a good bye?